eth/downloader/downloader.go

```diff
@@ -1500,7 +1500,8 @@ func (d *Downloader) importBlockResults(results []*fetchResult) error {
 	)
 	blocks := make([]*types.Block, len(results))
 	for i, result := range results {
-		blocks[i] = types.NewBlockWithHeader(result.Header).WithBody(result.Transactions, result.Uncles).WithWithdrawals(result.Withdrawals)
+		body := &types.Body{Transactions: result.Transactions, Uncles: result.Uncles}
```

Comment:
How do they do this upstream? It would definitely be cleaner to add a `func (r *result) Body() *types.Body` method (in a `libevm` file) for use throughout this file, but in the end we just want this to match what they do so all deltas eventually disappear.

Reply:
Indeed, they use a result `body() types.Body` method, almost right on point 👍